Plugin.php

```diff
@@ -45,14 +45,22 @@ class Plugin implements PluginInterface, EventSubscriberInterface
      */
     public function activate(Composer $composer, IOInterface $io)
     {
+        // Global assets repository
+        $repositoryManager = $composer->getRepositoryManager();
+        $repositoryManager->addRepository($repositoryManager->createRepository(
+            'composer',
+            ['url' => 'https://asset-packagist.org'],
+            'asset-packagist.org'
+        ));
```

Comment:
I don't think this part should be done.

Reply:
@samdark why?

Reply:
1) Implicit.
2) It fixes a problem that doesn't exist. Extensions have it already.

Reply:
@samdark 1. why need define repository in extensions that they don't use? 2. It's already have only in existed extensions of current project

Reply:
@WinterSilence the fact that you have a repository added to a package doesn't automatically add this repository to your main application. This change in the plugin does exactly that — always adding the repository no matter if it is used or not. I may choose to use fxp (yes, it's our past but still there are projects still using it directly) instead and then adding repository will not be desired.

Reply:
@samdark you still can use it https://getcomposer.org/doc/articles/repository-priorities.md#repository-priorities

Reply:
But you can not omit it anymore.

Reply:
@samdark you can't omit it's any way